opm/grid/cpgrid/processEclipseFormat.cpp

```diff
@@ -63,7 +63,7 @@
 namespace Dune
 {
 
-    using NNCMap = std::set<std::pair<int, int>>;
+    using NNCMap = std::set<std::pair<std::size_t, std::size_t>>;
```

Comment:
You have to change this in CpGridData.hpp as well, processEclipseFormat() declaration.